src/corewlan.cs

```diff
@@ -138,10 +148,25 @@ interface CWConfiguration : NSSecureCoding, NSMutableCopying {
 		[Static]
 		[Export ("configuration")]
 		CWConfiguration Create ();
+
+		[Mac (10,7), MacCatalyst (15,0)]
```

Comment:
We don't support 10.7 anymore, so that's redundant.

```suggestion
		[MacCatalyst (15,0)]
```